samples/matter/template/README.rst

```diff
@@ -51,8 +60,8 @@ Commissioning in Matter
 In Matter, the commissioning procedure takes place over Bluetooth LE between a Matter accessory device and the Matter controller, where the controller has the commissioner role.
 When the procedure has completed, the device is equipped with all information needed to securely operate in the Matter network.
 
-During the last part of the commissioning procedure (the provisioning operation), the Matter controller sends the Thread network credentials to the Matter accessory device.
-As a result, the device can join the Thread network and communicate with other Thread devices in the network.
+During the last part of the commissioning procedure (the provisioning operation), the Matter controller sends the Thread or Wi-Fi network credentials to the Matter accessory device.
```

Comment:
```suggestion
During the last part of the commissioning procedure (the provisioning operation), the Matter controller sends the network credentials to the Matter accessory device.
```
Maybe like this?

Reply:
I think it's important to emphasize that those are Thread or Wi-Fi credentials, while "network credentials" may suggest Matter network. I aligned the sentence to the one used by you in generic environment configuration description: https://github.com/nrfconnect/sdk-nrf/pull/8474/files#diff-917767b2147b7f8cdb4953de197f494b42b3fff0ebf0da84cca4853157e66f51R14